src/python/WMCore/WorkQueue/DataStructs/WorkQueueElement.py

```diff
@@ -71,8 +71,12 @@ def __init__(self, **kwargs):
         self.setdefault('OpenForNewData', False)
         # When was the last time we found new data (not the same as when new data was split), e.g. An open block was found
         self.setdefault('TimestampFoundNewData', 0)
-        # Should we check the location of the inputs, or trust the initial values?
+        # Trust initial input and pileup location or not
         self.setdefault('NoLocationUpdate', False)
```

Comment:
I'm in favor of removing it for any new WQE, though we still have to check whether old WQE have this flag enabled.